### PR TITLE
MULTIARCH-4696: MAC to multi-arch and use pre defined image

### PR DESCRIPTION
**Pull request by pkenchap**

### CODE REVIEW

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 1]
```diff
@@ -589,38 +529,35 @@ case "$CLUSTER_TYPE" in
...
   cleanup_ibmcloud_powervs "${CLEAN_VERSION}" "${WORKSPACE_NAME}" "${VPC_NAME}"
 
-  echo "Resource Group is ${RESOURCE_GROUP}"
+  echo "IC: Resource Group is ${RESOURCE_GROUP}"
   echo "${RESOURCE_GROUP}" > "${SHARED_DIR}"/RESOURCE_GROUP
 
   create_powervs_workspace "${WORKSPACE_NAME}"
-  create_powervs_private_network "${CRN}"
-  import_centos_image "${CRN}"
+  ic pi subnet create ocp-net --cidr-block 192.168.200.0/24 --net-type private --dns-servers 9.9.9.9 --gateway 192.168.200.1 --ip-range 192.168.200.10-192.168.200.250 --mtu 9100
```
[the remark this answers is not in the document]
Comment (pkenchap) [reply ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh]: Yeah sure , thanks added it .

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 2]
```diff
@@ -153,16 +140,17 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
 ibm_cloud_tgw              = "${workspace_name}-tg"
 EOF
 
-  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-mac-upi.tfvars "${SHARED_DIR}"/var-mac-upi.tfvars
-  cat "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-mac-upi.tfvars
+  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars "${SHARED_DIR}"/var-multi-arch-upi.tfvars
+  cat "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:144]:
```suggestion
  echo "UPI TFVARS created: ${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars
```
Let's not `cat` it out.

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 3]
```diff
@@ -153,16 +140,17 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
 }
 
 function create_upi_powervs_cluster() {
   cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ || true
   # Dev Note: https://github.com/ocp-power-automation/openshift-install-power/blob/devel/openshift-install-powervs#L767C1-L767C145
   # May trigger the redaction
   OUTPUT="yes"
-  ./openshift-install-powervs create -var-file var-mac-upi.tfvars -ignore-os-checks -verbose | sed '/.*client-certificate-data*/d; /.*token*/d; /.*client-key-data*/d; /- name: /d; /Login to the console with user/d' | \
+  /tmp/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:152]: Failure from https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/56018/rehearse-56018-periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le/1829439528125861888/build-log.txt is at this line... 


Try
```suggestion
  ${IBMCLOUD_HOME_FOLDER}/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 4]
```diff
@@ -178,12 +166,10 @@ function create_upi_powervs_cluster() {
         OUTPUT="yes"
         fi
     done || true
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:168]: Might not need the redaction with this change.

**Comment by prb112** [comment]
Left some comments

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 5]
```diff
@@ -153,16 +140,46 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+    fi
+
+    # build terraform from source using TERRAFORM_VERSION
+    echo "terraform is not installed, proceed to installing terraform"
+    cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ && curl -L https://github.com/hashicorp/terraform/archive/refs/tags/v"${TERRAFORM_VERSION}".tar.gz -o "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tar.gz \
+      && tar -xzf "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tar.gz && cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform-"${TERRAFORM_VERSION}" \
+      && go build -ldflags "-w -s -X 'github.com/hashicorp/terraform/version.dev=no'" -o bin/ . && cp bin/terraform "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform
+  fi
+  cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ || true
+  terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:181]:
```suggestion
  terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```
is terraform in the path?

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 6]
```diff
@@ -153,37 +140,73 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+    fi
+
+    # build terraform from source using TERRAFORM_VERSION
+    echo "terraform is not installed, proceed to installing terraform"
+    cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ && curl -L https://github.com/hashicorp/terraform/archive/refs/tags/v"${TERRAFORM_VERSION}".tar.gz -o "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tar.gz \
+      && tar -xzf "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tar.gz && cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform-"${TERRAFORM_VERSION}" \
+      && go build -ldflags "-w -s -X 'github.com/hashicorp/terraform/version.dev=no'" -o bin/ . && cp bin/terraform "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform
+  fi
+  echo "Applying terraform"
+  "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:185]:
```suggestion
  cd "${IBMCLOUD_HOME_FOLDER}"/ocp4-upi-powervs &&"${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 7]
```diff
@@ -153,37 +140,73 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+            if [[ "${LINE}" == *"END RSA PRIVATE KEY"* ]]
+            then
+            OUTPUT="yes"
+            fi
+        done
+  echo "POWERVS CLUSTER IS COMPLETE, log is redacted"
+  echo "Find ocp4-upi-powervs from root folder"
+  find /tmp/ -iname ocp4-upi-powervs
+  echo "Find all tfvars from root folder"
+  find /tmp/ -iname '*.tfvars'
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:206]:
```suggestion
```

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 8]
```diff
@@ -153,37 +140,73 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tfstate "${SHARED_DIR}"/terraform-multi-arch-upi.tfstate
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:207]:
```suggestion
  cp "${IBMCLOUD_HOME_FOLDER}"/ocp4-upi-powervs/terraform.tfstate "${SHARED_DIR}"/terraform-multi-arch-upi.tfstate
```

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 9]
```diff
@@ -153,37 +140,73 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+  "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform output -raw -no-color bastion_private_ip | tr -d '"' > "${SHARED_DIR}"/BASTION_PRIVATE_IP
+  "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform output -raw -no-color bastion_public_ip | tr -d '"' > "${SHARED_DIR}"/BASTION_PUBLIC_IP
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:209]:
```suggestion
  "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform output -raw -no-color bastion_private_ip | tr -d '"' > "${SHARED_DIR}"/BASTION_PRIVATE_IP
  "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform output -raw -no-color bastion_public_ip | tr -d '"' > "${SHARED_DIR}"/BASTION_PUBLIC_IP
```

make sure you have the right directory context.

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 10]
```diff
@@ -153,37 +140,69 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+    if [ -z "$(command -v go)" ]
+    then
+      echo "go is not installed, proceed to installing go"
+      cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ && wget -q https://go.dev/dl/go"${GO_VERSION}".linux-amd64.tar.gz && tar -C /tmp -xzf go"${GO_VERSION}".linux-amd64.tar.gz \
+        && export PATH=$PATH:/tmp/go/bin && export GOCACHE=/tmp/go && export GOPATH=/tmp/go && export GOMODCACHE="/tmp/go/pkg/mod"
+      if [ -z "$(command -v go)" ]
+      then
+        echo "Installed go successfully"
+      fi
+    fi
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:176]:
```suggestion
```

Comment (prb112) [reply ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:176]: I don't think we are using it.

Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:176]:
```suggestion
```

Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:176]:
```suggestion
```

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 11]
```diff
@@ -153,37 +140,69 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+
+    # build terraform from source using TERRAFORM_VERSION
+    echo "terraform is not installed, proceed to installing terraform"
+    cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ && curl -L https://github.com/hashicorp/terraform/archive/refs/tags/v"${TERRAFORM_VERSION}".tar.gz -o "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tar.gz \
+      && tar -xzf "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.tar.gz && cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform-"${TERRAFORM_VERSION}" \
+      && go build -ldflags "-w -s -X 'github.com/hashicorp/terraform/version.dev=no'" -o bin/ . && cp bin/terraform "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform
+  fi
+  echo "Applying terraform"
+  cd "${IBMCLOUD_HOME_FOLDER}"/ocp4-upi-powervs && "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:185]:
```suggestion
  cd "${IBMCLOUD_HOME_FOLDER}"/ocp4-upi-powervs && "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform apply -auto-approve -var-file "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars | \
```

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 12]
```diff
@@ -153,37 +140,69 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+            if [ ! -z "${OUTPUT}" ]
+            then
+                echo "${LINE}"
+            fi
+            if [[ "${LINE}" == *"END RSA PRIVATE KEY"* ]]
+            then
+            OUTPUT="yes"
+            fi
+        done
+  echo "POWERVS CLUSTER IS COMPLETE, log is redacted"
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:202]:
```suggestion
  echo "POWERVS CLUSTER IS COMPLETE, log is redacted"
  if [ ! -f "${IBMCLOUD_HOME_FOLDER}"/ocp4-upi-powervs/terraform.tfstate ]
  then
     echo "Terraform did not execute, exiting"
   fi
```

Comment (prb112) [reply ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:202]: you'll have to cleanup the formatting

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 13]
```diff
@@ -153,37 +140,61 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
...
+  cd "${IBMCLOUD_HOME_FOLDER}" \
+    && curl -L https://github.com/ocp-power-automation/ocp4-upi-powervs/archive/refs/heads/main.tar.gz -o "${IBMCLOUD_HOME_FOLDER}"/ocp.tar.gz \
+    && tar -xzf "${IBMCLOUD_HOME_FOLDER}"/ocp.tar.gz \
+    && mv "${IBMCLOUD_HOME_FOLDER}/ocp4-upi-powervs-main" "${IBMCLOUD_HOME_FOLDER}"/ocp4-upi-powervs
+  # short-circuit to download and install terraform
+  echo "Attempting to install terraform using gzip"
+  curl -L -o "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.gz -L https://releases.hashicorp.com/terraform/"${TERRAFORM_VERSION}"/terraform_"${TERRAFORM_VERSION}"_linux_amd64.zip \
+    && gunzip "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform.gz \
+    && chmod +x "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform \
+    && cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/terraform /usr/local/bin/ \
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:161]: creating link

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 14]
```diff
@@ -589,41 +555,37 @@ case "$CLUSTER_TYPE" in
...
   export PATH=$PATH:/tmp:/"${IBMCLOUD_HOME_FOLDER}"
 
-  setup_openshift_installer
   # Saving the OCP VERSION so we can use in a subsequent deprovision
   echo "${OCP_VERSION}" > "${SHARED_DIR}"/OCP_VERSION
 
-  setup_jq
   setup_ibmcloud_cli
 
   IBMCLOUD_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:563]: 00

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 15]
```diff
@@ -125,7 +112,7 @@ function create_upi_tf_varfile(){
   PULL_SECRET=$(<"${CLUSTER_PROFILE_DIR}/pull-secret")
   echo "${PULL_SECRET}" > "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/pull-secret.txt
 
-  cat <<EOF >${IBMCLOUD_HOME_FOLDER}/ocp-install-dir/var-mac-upi.tfvars
+  cat <<EOF >${IBMCLOUD_HOME_FOLDER}/ocp-install-dir/var-multi-arch-upi.tfvars
 ibmcloud_region     = "${POWERVS_REGION}"
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:116]: Right here is where the apiKey should go.

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Pull request merged**